build/tools/Automation/Services/FileService.cs

```diff
@@ -31,6 +31,11 @@ public void UpdateVersionsToBuildTxt(string platformName, string line)
             {
                 foreach (string debianFlavor in Constants.DebianFlavors)
                 {
+                    if (debianFlavor == "bookworm" && platformName == "python")
```

Comment:
@waliMSFT Just making sure this is on your radar, as once your 3.12 support is in hopefully we can remove this condition